### PR TITLE
feat(Label): new component

### PR DESCRIPTION
**Pull request by GaroGabrielyan**
### Chromatic Build Summary 
**Build URL:** https://www.chromatic.com/build?appId=66daee7859a843dd87bfb3d7&number=29 
**Storybook URL:** https://66daee7859a843dd87bfb3d7-rnuncafogj.chromatic.com/

### CODE REVIEW

File: src/lib/atoms/Label/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    isLoading,
+    className,
+    ...restProps
+}) => {
+    const labelRef = useRef<HTMLLabelElement | null>(null);
+
+    const isTruncated: boolean = useEllipsisDetection(labelRef);
+
+    return (
+        <div {...restProps} className={classnames('label', { className })}>
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.tsx:74]: remove classnames function

File: src/lib/atoms/Label/Label.test.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    });
+
+    it('renders htmlFor prop correctly', () => {
+        expect(setup.find('label').prop('htmlFor')).toStrictEqual(htmlFor);
+    });
+
+    it.each<ILabelProps['size']>(['medium', 'small'])('should have "%s" size', (size) => {
+        const wrapper = setup.setProps({ size });
+
+        expect(wrapper.props().size).toBe(size);
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.test.tsx:32]: please remove this test.

**Review by NairaMisakyan — DISMISSED** [review]

**Review by AregSoft — APPROVED** [review]

**Review by AregSoft — APPROVED** [review]

**Review by AregSoft — APPROVED** [review]

File: src/lib/atoms/Label/Label.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    /**
+     * Label size.<br/>
+     * Possible values: `medium | small`
+     */
+    size?: 'medium' | 'small';
+    /**
+     * The text content of the `label`.
+     * This is the main text displayed within the `label`.
+     */
+    children: string;
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.tsx:29]: change prop name!

File: src/lib/atoms/Label/Label.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        htmlFor: 'inputId',
+        size: 'medium',
+        children: 'label',
+        required: false
+    }
+};
+
+export default meta;
+
+const Template: FC<ILabelProps> = (args) => <Label {...args}>{args.children}</Label>;
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.stories.tsx:32]: `<Label {...args}/>`

File: src/lib/atoms/Label/Label.test.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ReactWrapper, mount } from 'enzyme';
+
+// Components
+import Label, { ILabelProps } from './index';
+import { Icon } from '../../../index';
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.test.tsx:6]: You use old  Icon version !

`import { IconName } from '@geneui/icons';`

File: src/lib/atoms/Label/Label.test.tsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    });
+
+    it('renders required prop correctly', () => {
+        const wrapper = setup.setProps({ required: true });
+        expect(wrapper.find('span').text()).toStrictEqual('*');
+    });
+
+    it('renders infoText prop correctly', () => {
+        const wrapper = setup.setProps({ infoText: 'text' });
+        expect(wrapper.find(Icon)).toBeTruthy();
```
Comment (NarekArshakyan) [inline src/lib/atoms/Label/Label.test.tsx:46]: its old Icon version !

**Review by AregSoft — APPROVED** [review]

**Review by NarekArshakyan — DISMISSED** [review]

**Review by NairaMisakyan — APPROVED** [review]

**Pull request merged**